### PR TITLE
RM-7494: The second postback of a double postbacks seems to be preferrably a button rather than a link

### PR DESCRIPTION
**Pull request by iam-mholle**
https://re-motion.atlassian.net/browse/RM-7494

### CODE REVIEW

**Review by Stefan-Ilic — APPROVED** [review]
I tested your change, it works 👍

File: Remotion/Web/Core/Res/HTML/SmartPage.js [hunk 1]
```diff
@@ -1176,11 +1176,20 @@ function SmartPage_Context(
...
+    {
+      if (_submitState.IsAutoPostback && !isAutoPostback)
+      {
+        _submitState.NextSubmitState = submitState;
+      }
+      else
+      {
+        _submitState.NextSubmitState = { CancelSubmit: true, Submitter: null };
+      }
+    }
```
Comment (MichaelKetting) [inline Remotion/Web/Core/Res/HTML/SmartPage.js:1192]: Please add an explation to the PR why this solution works. 

Originally, we've always overridden _submitState.NextSubmitState. Then, we've killed a second submits by overriding with CancelSubmit=true. Now, if a second submit is already queued, we just leave that one.

(right now, I don't want to re-think this myself)

Comment (iam-mholle) [reply Remotion/Web/Core/Res/HTML/SmartPage.js:1192]: Consider the following postback sequences:

### `Button`

- Set `_submitState`, nothing to queue here.

### `Autopostback`, `Button`

- Set `_submitState` for the autopostback.
- Set `_submitState.NextSubmitState` for the button click.

### `Button` (1), `Button` (2)

- Set `_submitState` for the first button.
- Set `_submitState.NextSubmitState` to `{ CancelSubmit: true, Submitter: null }` for the second button.

### `Autopostback`, `Link`, `Button`

- Set `_submitState` for the autopostback.
- Set `_submitState.NextSubmitState` for the link click.
- Button handling
  - Before:
    - Overwrite `_submitState.NextSubmitState`.
  - After:
    - Do nothing, there's already a queued postback.

**Review by MichaelKetting — APPROVED** [review]

**Comment by iam-mholle** [comment]
Added to master with 736a8e227d2501ae6bcfab8bee276b53c311cb01.

**Pull request closed without merging**